src/contract/nodes.rs

```diff
@@ -60,23 +63,41 @@ impl sha256t::Tag for NodeIdTag {
         sha256::HashEngine::from_midstate(midstate, 64)
     }
 }
-
-// TODO #48: Refactor all ids into a single style after `ConsignmentId`
-/// Unique node (genesis, extensions & state transition) identifier equivalent
-/// to the commitment hash
 #[cfg_attr(
     feature = "serde",
     derive(Serialize, Deserialize),
     serde(crate = "serde_crate")
 )]
 #[derive(
-    Wrapper, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Default, From,
-)]
-#[wrapper(
-    Debug, Display, LowerHex, Index, IndexRange, IndexFrom, IndexTo, IndexFull
+    Wrapper,
+    Copy,
+    Clone,
+    PartialEq,
+    Eq,
+    PartialOrd,
+    Ord,
+    Hash,
+    Default,
+    From,
+    Display,
 )]
+#[wrapper(Debug, LowerHex, Index, IndexRange, IndexFrom, IndexTo, IndexFull)]
+#[display(NodeId::to_bech32_string)]
 pub struct NodeId(sha256t::Hash<NodeIdTag>);
 
+impl bech32::Strategy for NodeId {
+    const HRP: &'static str = "node_id";
```

Comment:
We will use general LNPBP-14 HRP for all ids
```suggestion
    const HRP: &'static str = "id";
```